### DISCUSSION TITLE
angular material , not working right

### DISCUSSION THREAD

**Discussion by Victor-Nikiforov-Web** [storybookjs/storybook#17563] [Documentation] [answered]
i want to create angular library ,  it working fine but in story i cant see angular material styles and animations .
i did import all modules , buttons working fine but inputs not

**Comment by Karamellwuerfel** [comment, marked as answer]
Hi @Victor-Nikiforov-Web,
maybe you have to import the styles. Have a look at the Docs from storybook for material angular: https://storybook.js.org/recipes/@angular/material.
![grafik](https://github.com/storybookjs/storybook/assets/23174276/defdc558-f3f0-44f9-b878-21a3bc445349)
